charts/celery-exporter/Chart.yaml

```diff
@@ -12,4 +12,4 @@ sources:
 maintainers:
   - name: danihodovic
   - name: adinhodovic
-version: 0.7.0
+version: 0.7.1
```

Comment:
Bumped because [the `test/helm-chart` job is failing](https://github.com/danihodovic/celery-exporter/actions/runs/11378040913/job/31688727794) due no version bump detected despite the removal of `appVersion` in https://github.com/danihodovic/celery-exporter/pull/329/commits/b697d34ace08f9bd18cc23db50280238c27ae36e ...

<img width="1556" alt="Screenshot 2024-10-17 at 21 03 46" src="https://github.com/user-attachments/assets/9705a4d1-489a-4ebb-9c46-a8b6b114b8b0">